src/identity/interaction/util/getFormDataRequestBody.ts

```diff
@@ -0,0 +1,33 @@
+import type { ParsedUrlQuery } from 'querystring';
+import { parse } from 'querystring';
+import type { HttpRequest } from '../../../server/HttpRequest';
+import { UnsupportedMediaTypeHttpError } from '../../../util/errors/UnsupportedMediaTypeHttpError';
+
+/**
+ * Architecturally, this file doesn't make any sense. It simply exists in this form
```

Comment:
It's fine; let's just comment what it does, and rename this file to `FormDataUtil.ts`.

Reply:
Done